### ISSUE TITLE
v2

### ISSUE THREAD

**Issue by retokromer** [FFmpeg/FFV1#120]
I suggest to add to the specification the reference where the never released version 2 can be found.

**Comment by retokromer** [comment]
Visibly not of interest, considering that the [version published yesterday](https://datatracker.ietf.org/doc/html/draft-ietf-cellar-ffv1-v4-01) ignores it and that nobody commented.

**Comment by michaelni** [comment]
A patch adding a reference is welcome.

**Comment by retokromer** [comment]
For historical reasons, I think it may be of interest to know that [FFV1.2](https://github.com/FFmpeg/FFV1/blob/8ad772b6d61c3dd8b0171979a2cd9f11924d5532/ffv1.lyx) did exist, even if never officially released. And where one interested in can find it. I am not at all suggesting to include the 2 specs into the standard candidate, just the link to the source.

(FYI: I did actually see it, when you published it back in 2012, and I also have played with as I told @pjotrek-b in Singapore during a pleasant evening.)

**Comment by pjotrek-b** [comment]
...but FFV1.2 was never released as "non-experimental" and directly superseded by FFV1.3 :smile:

**Comment by retokromer** [comment]
I know very well, but as experimental the format actually did exist! And it may be of interest to be able to follow «easily» the different steps of this evolution.

BTW: I checked, I have a small set of exactly 12 historic files which I did generate with a film scanner for small gauge in both 1 and 2 in order to check the improvements made. 😄😄 Yet, as I told you in Singapore, the bigger part of our video production was in HuffYUV at that time and only a smaller part in FFV1 (0, 1 and 3). After our big data migration in 2014 from LTO-4 to LTO-6, we internally do store all stream-based content in FFV1 version 3.
